project/VersionUtil.scala

```diff
@@ -79,9 +79,8 @@ object VersionUtil {
             // Workaround lack of git worktree support in JGit https://bugs.eclipse.org/bugs/show_bug.cgi?id=477475
             val sha = List("git", "rev-parse", "HEAD").!!.trim
             val commitDateIso = List("git", "log", "-1", "--format=%cI", "HEAD").!!.trim
-            val date = java.util.Date.from(DateTimeFormatter.ISO_DATE_TIME.parse(commitDateIso, new TemporalQuery[Instant] {
-              override def queryFrom(temporal: TemporalAccessor): Instant = Instant.from(temporal)
-            }))
+            val date = java.util.Date.from(DateTimeFormatter.ISO_DATE_TIME.parse
+            (commitDateIso, (temporal: TemporalAccessor) => Instant.from(temporal)))
```

Comment:
Clever! It is so compact that it would fit on a single line.

Reply:
@som-snytt Can you please merge this change.

Reply:
It's not currently mergeable, as the code is badly indented. I believe that's what Som is alluding to with his remark. The second line is a continuation of the first, so it shouldn't be at the same indentation level.

There are many possible ways to indent this code — we don't enforce a particular style in this repo — but if you don't have a favorite yourself, one way to fix it would be:

```scala
            val date = java.util.Date.from(DateTimeFormatter.ISO_DATE_TIME.parse(commitDateIso,
              (temporal: TemporalAccessor) => Instant.from(temporal)))
```

Som's apparent suggestion of just jamming the entire thing onto one line isn't my favorite way to go here, but it would also be acceptable.

Reply:
```
val date = Date.from(DateTimeFormatter.ISO_DATE_TIME.parse(commitDateIso, Instant.from(_)))
```
`Date` is already imported.

Reply:
ah, with those changes, it indeed fits satisfactorily on one line.

Reply:
could also just import `ISO_DATE_TIME` for more terseness.

Reply:
also remove the unneeded import of temporal things.

Reply:
@som-snytt ISO_DATE_TIME is already imported there as import java.time.format.DateTimeFormatter. 
Also,  unneeded import of temporal things are removed. We can merge it now.

Reply:
The indentation is still bad